Common/Tests/Utilities/AssertUtil.cs

```diff
@@ -199,14 +199,17 @@ public static void DoublesEqual(double expected, double actual, string error)
 
         [System.Diagnostics.DebuggerStepThrough]
         public static void DoesntContain<T>(IEnumerable<T> source, IEnumerable<T> value) {
+            var contains = new List<T>();
```

Comment:
value.Any(v => source.Any(s => s.Equals(v)) ? Up to you. Linq is hard to debug ;-)

Reply:
That also doesn't give back the items that cause the test to fail. So `source.Intersect(value).ToArray()` is probably what's needed, but it's no big deal here. This is just tests